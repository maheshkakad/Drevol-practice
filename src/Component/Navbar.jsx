import React from 'react'

const Navbar = () => {
  return (
    <div>
         <h1>mahesh kakad</h1> 
          
          <ul>
             <li>m</li>
             <li>ds</li>
             <li>dsd</li>
             <li>dsd</li>
             <li>dsd</li>
          </ul>
    </div>
  )
}

export default Navbar