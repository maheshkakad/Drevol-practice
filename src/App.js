
import { useState } from 'react';
import './App.css';
import Clickers from './Component/Clickers';
import Navbar from './Component/Navbar';
import Button from './Component6/Button';
import Input from './Component6/Input';
import Users from './Component6/Users';

function App() {

  const [showDiv , setShowDiv ] = useState("");

  let a = 5;
  let b = 5;
  return (
    <div className="App">
      {/* <Navbar />
        <h1>mahesh</h1>
        <h4>parent</h4>

        <h3 data-testid="mytestid"> hello there</h3>

         <div title="sum">{a+b}</div>


         //Clickers

         <Clickers /> */}
  
   {/* <Input showDiv={showDiv} />

   <Button /> */}
   <Users />

    </div>
  );
}

export default App;
