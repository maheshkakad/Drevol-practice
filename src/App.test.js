import { render, screen } from '@testing-library/react';
import App from './App';
import Navbar from './Component/Navbar';
import { RenderResult } from '@testing-library/react';
import renderer from 'react-test-renderer'

test('renders learn react link', () => {
  render(<Navbar />);
  const linkElement = screen.getByText(/mahesh/i);
  expect(linkElement).toBeInTheDocument();
});


test('renders learn react link', () => {
  render(<Navbar />);
  const linkElement = screen.getAllByRole("listitem");
  expect(linkElement.length).toBe(5)
});

// test('render react component' , ()=> {
//    render(<App />)
//    const  elementapp = screen.getByText(/parent/i)
//    expect(elementapp).toBeInTheDocument()
// })

// test('render react component' , ()=> {
//   render(<App />)
//  const  elementapp = screen.getByTestId("mytestid")
//   expect(elementapp).toBeInTheDocument()
// })


// test('render react title' , ()=> {
//   // let renderResult:RenderResult;
 
//    const component  = renderer.create( <App />);

//    let tree = component.toJSON()

//    expect(tree).toMatchSnapshot();

// })
