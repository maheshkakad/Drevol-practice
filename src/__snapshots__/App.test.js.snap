// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render react title 1`] = `
<div
  className="App"
>
  <div>
    <h1>
      mahesh kakad
    </h1>
    <ul>
      <li>
        m
      </li>
      <li>
        ds
      </li>
      <li>
        dsd
      </li>
      <li>
        dsd
      </li>
      <li>
        dsd
      </li>
    </ul>
  </div>
  <h1>
    mahesh
  </h1>
  <h4>
    parent
  </h4>
</div>
`;
